src/price_data.py

```diff
@@ -563,3 +568,45 @@ def get_cost(
         if isinstance(tr, transaction.SoldCoin):
             return price * tr.sold
         raise NotImplementedError
+
+    def check_database(
+        self,
+    ):
+        for db_path in Path(config.DATA_PATH).glob("*.db"):
+            if db_path.is_file():
+                platform = db_path.stem
+
+                try:
+                    get_price = getattr(self, f"_get_price_{platform}")
+                except AttributeError:
+                    if platform == "coinbase":
+                        get_price = self._get_price_coinbase_pro
+                    else:
+                        raise NotImplementedError(
+                            "Unable to read data from %s", platform
+                        )
+
+                with sqlite3.connect(db_path) as conn:
+                    query = "SELECT name FROM sqlite_master where type='table'"
+                    cur = conn.execute(query)
+
+                    for table in cur.fetchall():
+                        tablename = table[0]
+                        pair = tablename.split("/")
+                        query = f"SELECT utc_time FROM `{tablename}` WHERE price<=0.0;"
+                        cur = conn.execute(query)
+
+                        for row in cur.fetchall():
+                            utc_time = datetime.datetime.strptime(
+                                row[0], "%Y-%m-%d %H:%M:%S%z"
+                            )
+                            price = get_price(pair[0], utc_time, pair[1])
+                            log.warning(
```

Comment:
This shouldn't be a warning, merely an info, because this is exactly what the function is good for.

But we could warn at the end, if and how many rows are still `price<=0.0`, perhaps separated by platforms (total: 10 rows, binance: 5 rows, ...)

Reply:
log.info does not show by default it seems.